### ISSUE TITLE
Need a way for Quasar to honor `npm install` lifecycle hooks during production build process

### ISSUE THREAD

**Issue by Cure4Life** [quasarframework/quasar#9672]
**Is your feature request related to a problem? Please describe.**
In my project's package.json I need to run a postinstall hook to rebuild sqlite3 for electron using electron-rebuild. I do this by using the postinstall lifecycle hook in the scripts section. 

`"postinstall": "electron-rebuild -f -w sqlite3",`

Everything is fine in DEV mode but when building for production, Quasar is running "npm i --production" which is installing a fresh sqlite3,  

`App • Running "npm install --production`

This causes the resulting app to crash upon trying to open the database since sqlite3 is not getting rebuilt for electron.

**Describe the solution you'd like**
Have Quasar include any lifecycle hooks in the project's main package.json scripts section to be executed when running "npm install --production"

**Describe alternatives you've considered**
Maybe some hook could be added to quasar.conf.js or an option to indicate which hooks in the original package.json to include

**Additional context**
I was hoping to convert a project I recently started using React / Electron over to using Quasar but this may prevent me from doing that. Any suggestions would be much appreciated!

**Comment by Cure4Life** [comment]
Oh what a good night's sleep can do! I believe I have answered my own question - by switching from electron-packager to electron-builder in quasar.conf.js. Turns out, electron-builder automatically rebuilds native dependencies as part of the normal build process by default.

https://www.electron.build/configuration/configuration
`npmRebuild = true Boolean - Whether to rebuild native dependencies before starting to package the app.`
